### PR TITLE
Move Coupler to cirq_gooogle.ops

### PR DESCRIPTION
**Pull request by dstrain115**
- Noticed when adding code that there were circular imports due to Couplers api codes needing code from devices.
- Moved this to ops directory instead, which avoids the circular import.
- Note that, although GridQubit is in cirq.devices, LineQubit and NamedQubit are in cirq.ops, so this also seems to be a good place to put this class.

### CODE REVIEW

File: cirq-google/cirq_google/devices/grid_device.py [hunk 1]
```diff
@@ -37,8 +37,8 @@
 from cirq_google import transformers
 from cirq_google.api import v2
 from cirq_google.devices import known_devices
-from cirq_google.devices.coupler import Coupler
 from cirq_google.experimental import ops as experimental_ops
+from cirq_google.ops.coupler import Coupler
```
Comment (pavoljuhas) [inline cirq-google/cirq_google/devices/grid_device.py:41]: For the sake of consistency with `ops.Stuff` syntax at 49-52, can we drop this import and use `ops.Coupler` below?

**Review by pavoljuhas — APPROVED** [review]
LGTM with 2 nitpicky comments.

File: cirq-google/cirq_google/api/v2/program.py [hunk 2]
```diff
@@ -77,7 +72,6 @@ def qubit_from_proto_id(proto_id: str) -> cirq.Qid:
         A `cirq.Qid` corresponding to the proto id.
     """
     # Avoid circular import
```
[the remark this answers is not in the document]
Comment (dstrain115) [reply cirq-google/cirq_google/api/v2/program.py]: Done.

File: cirq-google/cirq_google/devices/grid_device.py (hunk 1 above)
Comment (dstrain115) [reply cirq-google/cirq_google/devices/grid_device.py]: Added as cg_ops (it's cirq_google.ops)

File: cirq-google/cirq_google/devices/grid_device.py [hunk 3]
```diff
@@ -38,7 +38,7 @@
 from cirq_google.api import v2
 from cirq_google.devices import known_devices
 from cirq_google.experimental import ops as experimental_ops
-from cirq_google.ops.coupler import Coupler
+from cirq_google import ops as cg_ops
```
Comment (pavoljuhas) [inline cirq-google/cirq_google/devices/grid_device.py:41]: Let's then delete line 36 and only use `cg_ops` for that module.

**Review by pavoljuhas — CHANGES_REQUESTED** [review]
Let us use only one of `cg_ops`, `ops` for `cirq_google.ops`.

File: cirq-google/cirq_google/devices/grid_device.py (hunk 3 above)
Comment (dstrain115) [reply cirq-google/cirq_google/devices/grid_device.py]: Ah, missed that this was cirq_google.ops (thought it was cirq.ops).  Reverted this and changed to ops.Coupler for brevity.

**Review by pavoljuhas — APPROVED** [review]

**Pull request merged**